.github/workflows/release-production.yml

```
@@ -64,3 +64,26 @@ jobs:
           name: ${{ env.PACKAGE_NAME }}-v${{env.TAG}}
           path: ./*.tgz
           if-no-files-found: error
+
+  helm-chart-release:
+    runs-on: ubuntu-latest
```

Could you add a condition to make the job not run until we complete moving branches.
I want you PR to be able to go in without you getting more conflicts with other changes.
The multi chart structure in particular is a very good contribution I don't want to hold up.
```suggestion
    if: false
    runs-on: ubuntu-latest
```